native/src/utils/__tests__/sentry.spec.ts

```diff
@@ -106,7 +106,7 @@ describe('log', () => {
     expect(Sentry.addBreadcrumb).toHaveBeenCalledTimes(1)
     expect(Sentry.addBreadcrumb).toHaveBeenCalledWith({
       message: 'test error log',
-      level: Sentry.Severity.Error
+      level: 'error'
```

Comment:
See https://github.com/getsentry/sentry-javascript/blob/7.0.0/MIGRATION.md#severity-severitylevel-and-severitylevels.
Sentry.Severity enum does not exist anymore, instead just plain strings should be used. Typescript makes sure that only valid values are passed. Therefore it is even more robust than before since it was possible to use invalid severity levels.